app/assets/javascripts/XMLDisplay.js

```diff
@@ -0,0 +1,195 @@
+/* Copyright (c) 2007 Lev Muchnik <LevMuchnik@gmail.com>. All rights reserved.
+* You may copy and modify this script as long as the above copyright notice,
+* this condition and the following disclaimer is left intact.
+* This software is provided by the author "AS IS" and no warranties are
+* implied, including fitness for a particular purpose. In no event shall
+* the author be liable for any damages arising in any way out of the use
+* of this software, even if advised of the possibility of such damage.
```

Comment:
Please also include this copyright notice in the LICENSES.txt file in the "Third Party" section.